Bugzilla/Template.pm

```diff
@@ -771,11 +771,17 @@ sub create {
         1
       ],
 
-      bug_list_link => sub {
-        my ($buglist, $options) = @_;
-        return
-          join(", ", map(get_bug_link($_, $_, $options), split(/ *, */, $buglist)));
-      },
+      bug_list_link => [
+        sub {
+          my ($context, $options) = @_;
+          return sub {
+            my $buglist = shift;
+            return join(", ",
+              map(get_bug_link($_, $_, $options), split(/ *, */, $buglist)));
```

Comment:
```suggestion
              map { get_bug_link($_, $_, $options) } split(/\s*,\s*/, $buglist));
```

only ever use the block form of `grep` and `map` and friends. The other one is confusing and weird. I mean, more than normal.

Also literal spaces in regex are bad, editors can screw that up and things like `\t` can be hard to find bugs.

Reply:
I think I copy & pasted from somewhere else. Thanks!

Reply:
Yeah. it's safe to say a lot of existing code wouldn't pass my code review.